cmd/gb/env.go

```diff
@@ -13,8 +13,12 @@ func init() {
 
 var EnvCmd = &cmd.Command{
 	Name:      "env",
-	ShortDesc: "env prints the project environment variables",
-	Run:       env,
+	UsageLine: `env`,
+	Short:     "print project environment variables",
+	Long: `Env prints project environment variables.
+
+`,
+	Run: env,
```

Comment:
Yeah. It's not awesome, but it sort of works for now and closes #152.

On Mon, Jun 1, 2015 at 6:53 PM, Dávid Kaya <notifications@github.com> wrote:

> In cmd/gb/env.go
> <https://github.com/constabulary/gb/pull/169#discussion_r31408963>:
>
> > @@ -13,8 +13,12 @@ func init() {
> >
> >  var EnvCmd = &cmd.Command{
> >  	Name:      "env",
> > -	ShortDesc: "env prints the project environment variables",
> > -	Run:       env,
> > +	UsageLine: `env`,
> > +	Short:     "print project environment variables",
> > +	Long: `Env prints project environment variables.
> > +
> > +`,
> > +	Run: env,
>
> Oh ok.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/constabulary/gb/pull/169/files#r31408963>.
>